crt_portal/cts_forms/templates/forms/pro_template.html

```diff
@@ -16,10 +16,13 @@
       </section>
     {% endif %}
 
-    <h2 id="intake">Intake</h2>
-    {% include "forms/question_cards/date.html" with question='CRT receipt date' month=form.crt_reciept_month day=form.crt_reciept_day year=form.crt_reciept_year required=False %}
-
-    {% include "forms/question_cards/single_question.html" with field=form.intake_format %}
+    <h2 class="h1" id="intake">Intake</h2>
```

Comment:
`h2 class="h1"` ... :cold_sweat: I think it would be easier just to make these `h1`s. But I realize this may be out of scope.

Reply:
omg, styles like this are honestly one of my biggest pet peeves! But there are already `h1`s on these pages and I'm still not super clear on what the accessibility implications of multiple h1s on a page are 😬. I could pretty easily make a class like `section-title` that inherits the right styles, though, which would be a little less skeevy